### PR TITLE
Update cass-config-builder and system-logger images

### LINKED ISSUE #1531: Update support images
## Bug Report

**Describe the bug**
cass-config-builder and system-logger are both outdated versions. Update them to current ones. 

**To Reproduce**

**Expected behavior**

**Screenshots**

**Environment (please complete the following information):**

* Helm charts version info

`$ helm ls -A` 

* Helm charts user-supplied values

`$ helm get values RELEASE_NAME` 

* Kubernetes version information:

`kubectl version`

* Kubernetes cluster kind:

**Additional context**

### PR DESCRIPTION
**Pull request by burmanm**
**What this PR does**:
Updates cass-config-builder, system-logger images and cass-operator to 1.13.1

**Which issue(s) this PR fixes**:
Fixes #1531 

**Checklist**
- [ ] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [x] CHANGELOG.md updated (not required for documentation PRs)
- [x] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

**Comment by jsanda** [comment]
How come tests didn't run?

**Comment by adejanovski** [comment]
Not sure. This has happened a few times lately 🤔

File: charts/k8ssandra-operator/Chart.yaml [hunk 1]
```diff
@@ -3,14 +3,14 @@ name: k8ssandra-operator
 description: |
   Kubernetes operator which handles the provisioning and management of K8ssandra clusters.
 type: application
-version: 0.38.4
+version: 0.38.5
```
Comment (adejanovski) [inline charts/k8ssandra-operator/Chart.yaml:6]: You'll need to rebase and increment the version again as main is already on `0.38.5` for the `1.4.0` release.

**Review by adejanovski — APPROVED** [review]
Looks good, but needs a rebase.

**Comment by burmanm** [comment]
Replaced by #1543

**Pull request closed without merging**